src/azure-cli/azure/cli/command_modules/resource/commands.py

```diff
@@ -459,15 +453,7 @@ def load_command_table(self, _):
         g.custom_show_command('show', 'cli_managementgroups_group_show')
         g.custom_command('create', 'cli_managementgroups_group_create')
         g.custom_command('delete', 'cli_managementgroups_group_delete')
-        g.generic_update_command(
-            'update',
-            getter_name='cli_managementgroups_group_update_get',
-            getter_type=resource_managementgroups_update_type,
-            setter_name='cli_managementgroups_group_update_set',
-            setter_type=resource_managementgroups_update_type,
-            custom_func_name='cli_managementgroups_group_update_custom_func',
-            custom_func_type=resource_managementgroups_update_type,
-            exception_handler=managementgroups_exception_handler)
+        g.custom_command('update', 'cli_managementgroups_group_create')
```

Comment:
@git-thomasdolan In fact, `cli_managementgroups_group_update_set` is different from `cli_managementgroups_group_create`. `cli_managementgroups_group_create` is a `PUT` request and `cli_managementgroups_group_update_set`  is a `PATCH` request, which may lead to breaking change

Reply:
Is there a simple way to change the JSON parameter name in the request body when it gets to the REST service. I found that when upgrading from 0.2.0 to 1.0.0 the parameter "parentId" changed to "parentGroupId." Our code only recognizes "parentId" and since the python SDK sends it as "parentGroupId" it is not being recognized. If this requires updating the python SDK or causes a breaking change in the CLI then we can just fix it in our CodeBase. I was just curious to see if there is a quick fix that can be made here.

Reply:
@git-thomasdolan Since CLI sends a request to the REST service through Python SDK, we need to modify the SDK first, and then modify the incoming parameters of the SDK in CLI

Reply:
@git-thomasdolan This PR can't catch up with the release of this sprint. We'll postpone it to the next sprint (03-01). Do you think this date is acceptable?

Reply:
@zhoxing-ms Yes, March sprint is fine. It also gives us more time to finish the additional functionality in the managementgroup CLI extension and remove the duplicate commands. For now, ManagementGroups is making a change to accept both "parentId" and "parentGroupId" for incoming requests. I do not think we will have to make a change to the python SDK.